server/routes/api/legislators/find-by-district.js

```diff
@@ -12,9 +13,32 @@ var get = function (req, res) {
 
   if (!Congress.validDistrict(state, district))
     res.status(400).json(resHelpers.makeError({ message: "Bad value for state/district parameter" }));
-  else
-    res.json(resHelpers.makeResponse(Congress.getLegislators(state, district)));
-};
+  else {
+    var legislators = Congress.getLegislators(state, district);
+    var bioguideIds = legislators.map(function(legislator) {
+      return legislator.bioguideId;
+    });
+    
+    if (bioguideIds && legislators) {
+      // Call the PotC API get defunct and contact_url properties for each legislator
+      potc.getFormElementsForRepIdsFromPOTC(bioguideIds, req.app.locals.CONFIG, function(err, data) {
+        if (err) {
+          res.status(400).json(resHelpers.makeError(err));
```

Comment:
I think we need to return here due to the bug from #97 (the call to `res.status(...).json` will make the subsequent call to `res.json` on L39 blow up).